.gitignore

```diff
@@ -9,3 +9,6 @@ third_party/spirv-headers
 
 # Vim swap files
 [._]*.s[a-w][a-z]
+
+# Android
+android_sample/build/
```

Comment:
Why does this need it's own build folder? It should just use what the user selects, no?

Reply:
Updated. I just let user decide the build folder.